shiksha-api/app-service/app/services/general_chat_service.py

```
@@ -47,12 +48,19 @@ def __init__(self):
             )
 
 
+    @observe(name="Shiksha-QA")
     @validate_call
     async def __call__(
```

Ruff (0.15.18)

[warning] Missing return type annotation for special method `__call__`

(ANN204)